### ISSUE TITLE
PubSub with Socket.get via BluePrint does not subscribe client to any event (Autowatch is true)

### ISSUE THREAD

**Issue by adhambadr** [balderdashy/sails#2794]
I wanted to build a quick BluePrint Publish subscribe but it didn't work out. 
The default find and FindOne of the Blueprint does not automatically subscribe the request if its a socket to the model. 
Either its a bug or the docs http://sailsjs.org/#!/documentation/reference/websockets/resourceful-pubsub and http://sailsjs.org/#!/documentation/reference/websockets/resourceful-pubsub/watch.html are outdated. 
Sails v 0.11.0, created a new controller, model (User) with only name attribute, then just like the example in the doc called : 

`io.socket.on("user", function(event){console.log(event);})`
`io.socket.get("/user", function(resData, jwres) {console.log(resData);})`

Console outputs current list of available instances but nothing happens on create or update from any other external source
I call the blueprint /user/create and /user/update/1
config.blueprints.autoWatch is true, mirror is true. 

I tested if the issue is on the create and update side if the client explicitly calls subscribe or watch, update and create events are successfully triggered and caught. 
However according to the example in this docs http://sailsjs.org/#!/documentation/reference/websockets/resourceful-pubsub/subscribe.html you could use subscribe to listen to new creates, which I think is not the case anymore. As I understood its now either watch() for create events, and subscribe for anything else. 
Eitherways calling the default Blueprint GET "/model/get" via a socket does not subscribe to either.

**Comment by cruzlutor** [comment]
is this still happend?

**Comment by toddgeist** [comment]
I see this same behavior. 

`io.socket.get("</modelname>", function(resData, jwres) {console.log(resData);})` does not subscribe to anything with just default blueprints on. I have to override the get route and manually subscribe to get it to work.

**Comment by toddgeist** [comment]
follow up:  
It appears that you have to add `autoSubscribe:true` to your model for the subscriptions to occur.  You can add it in the individual modal or in config/modals.js

**Comment by sailsbot** [comment]
Thanks for posting, @adhambadr.  I'm a repo bot-- nice to meet you!

It has been 60 days since there have been any updates or new comments on this page.  If this issue has been resolved, feel free to disregard the rest of this message.  On the other hand, if you are still waiting on a patch, please:

  + review our [contribution guide](https://github.com/balderdashy/sails/blob/master/CONTRIBUTING.md) to make sure this submission meets our criteria (only _verified bugs_ with documented features, please;  no questions, commentary, or bug reports about undocumented features or unofficial plugins)
  + create a new issue with the latest information, including updated version details with error messages, failing tests, and a link back to [the original issue]().  This allows GitHub to automatically create a back-reference for future visitors arriving from search engines.

Thanks so much for your help!
